Stylll/Book-A-Meal
# #159718328 improve code quality

#### What does this PR do?
Improve overall code quality of the codebase

#### Description of Task to be completed?
- update variable and file names
- add more test cases to increase client side coverage
- update test cases to be more descriptive

#### How should this be manually tested?
- Go through the codebase to review variable names and file names
- run `npm run test:client` to see the client side coverage percentage
- check through the client test suite for updated test cases and additional test cases

#### What are the relevant pivotal tracker stories?
#159718328

#### Any background context you want to add?

#### Screenshots
